pkgs/os-specific/darwin/whatsapp/default.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, fetchzip
+, stdenvNoCC
+, makeWrapper
+}:
+
+stdenvNoCC.mkDerivation rec {
```

```suggestion
stdenvNoCC.mkDerivation (finalAttrs: {
```

Using `rec` is discouraged

https://nix.dev/guides/best-practices#recursive-attribute-set-rec